helm/content-store/templates/deployment.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ .Release.Name }}
+  labels:
+    app: {{ .Release.Name }}
+spec:
+  replicas: {{ .Values.deployment.replicaCount }}
```

Comment:
yeah, I saw the guidance on helm but Nadeem thought it is easier to follow this way. It's a big change so we shall see.

Reply:
I made the changes now. I think it can easily be confusing whether to nest or not if you see the generated values.yaml from helm create: deployment is not tested at the top level (might be to do with containerPort being reused elsewhere) but ingress is nested as well as service.